sass/elements/_html.scss

```diff
@@ -11,8 +12,21 @@ html, body {
     background-color: #232326;
 }
 
+div.incorrect {
+    background-color: #542326;
+    border-left: 10px solid red;
+    border-radius: 10px;
+
+    .z-code,
+    .z-code code {
+        background-color: #542326;
+    }
+}
+
```

Comment:
I think is better to put this snippet in `components/_syntax-theme.scss` which is where all the syntax highlighting  styles are. This file (`html.scss`) should be only for `html` element tweaks (although it also touches `body` right now, this is fixed in #315).

Reply:
ok, I've moved it~